### ISSUE TITLE
Proposal: Onchain Web3 Analysis

### ISSUE THREAD

**Issue by Latsan** [kaiachain/kaia-dapp-mono#285]
Description
A weekly Web3 analytics report focusing on Kaia ecosystem metrics, including transaction volumes, active addresses, gas usage, and DeFi/NFT market trends. The analysis will leverage SQL querying skills with Dune/Flipside Analytics to provide actionable insights for the community. Reports will include:
- On-chain activity analysis and visualization
- DeFi protocol performance metrics
- NFT market trends and volume analysis
- User adoption patterns and growth metrics

Social Links:
Twitter: https://x.com/latsan04
Telegram: https://t.me/latsan04

Bounty track
- [Web3 Analysis]

**Comment by zxstim** [comment]
Can you use Dune? we are missing a few Dune boards

**Comment by Latsan** [comment]
> Can you use Dune? we are missing a few Dune boards

Yes, I can use Dune very well. My username is Latsan on Dune and I have a couple of nice dashboards there.

**Comment by zxstim** [comment]
cool. Come back with a dashboard or a few to claim bounty

make BUIDL on dorahacks too 
https://dorahacks.io/hackathon/kaia-winter-epoch/detail#submission-guidelines

**Comment by Latsan** [comment]
Gm
The analysis is ready and open source. I can extract all the codes and upload to the KAIA github repo
There dashbaord is over here - https://dune.com/latsan/kaia-dex-competition-and-market-share-analysis

**Comment by Latsan** [comment]
Hi there @zxstim 
Just a temp check

**Comment by zxstim** [comment]
just got back from vacation

> I can extract all the codes and upload to the KAIA github repo

no need but you can make a guide on how to use Dune that will be counted as a separate grant

Approved for `1200 KAIA with 50% penalty = 600 KAIA` (reason: https://x.com/KaiaDevIntern/status/1866841240433725729)

builder community: https://x.com/i/communities/1867063240926302471

Get active on X, make an update issue later to get the 50% penalty back

**Comment by zxstim** [comment]
closing

**Comment by zxstim** [comment]
What's your Dorahacks BUIDL? (https://dorahacks.io/hackathon/kaia-winter-epoch/detail#submission-guidelines)

**Comment by Latsan** [comment]
Hi @zxstim 
Thank you for the warm feedback, I will be active on twitter creating contents for the analysis.
AlsoI will know which area do you feel I should dive into to produce more in depth analysis for KAIA


Acknowledgement: I confirm the below address is my bounty payout address.
Payout address: 0x74314BbD12D7BaC61518b741B4547eA06ac08D06

**Comment by zxstim** [comment]
What's your Dorahacks BUIDL? can't pay if you don't make one

**Comment by Latsan** [comment]
here https://dorahacks.io/buidl/21339 mate

**Comment by zxstim** [comment]
Bro, you have to follow the submission guideline and have the wallet inside the BUIDL

https://dorahacks.io/hackathon/kaia-winter-epoch/detail#submission-guidelines

i don't want to be rude but the guidelines are there for a reason

**Comment by Latsan** [comment]
So sorry about that, I mis-interpreted the guideline
Fixed now

**Comment by Latsan** [comment]
Payment received
Thanks mate 

Already promoting the dashboard on X gradually so I can be of 50% penalty
Thanks once again

**Comment by zxstim** [comment]
make a new issue update, add more dashboards and cool idea (use the update issue template)

the penalty will be granted with the payout for that issue
